spec/acceptance/follow_ups_controller_spec.rb

```diff
@@ -2,14 +2,14 @@
 
 require 'acceptance_helper'
 
-resource 'Follow Ups' do
+resource 'FollowUps' do
   header 'Accept', 'application/vnd.api+json'
   header 'Content-Type', 'application/vnd.api+json'
   let(:raw_post) { params.to_json }
 
   post 'follow_ups/' do
     let(:data) do
-      { type: :follow_up,
+      { type: type,
```

Comment:
I'm not seeing where `type` is defined like in the other specs? (inexperienced rails developer question)

Reply:
`type` is a helper (from the acceptance spec helper) and uses a naming convention: 
https://github.com/CruGlobal/mobile-content-api/pull/181/commits/c554d0cc3aa55e508dda2c5bd3e74bf2abe37412#diff-4d353319af1985fd16aee40f0bbb5e6cR25

... of course any explicit `let(:type) { 'some-name'' }` will override

Reply:
moved that commit to https://github.com/CruGlobal/mobile-content-api/pull/179 (as I was fixing coverage)
... so once that is merged I will rebase here

Reply:
Was guessing it was my lack of rails experience :)